styles.css

```
@@ -0,0 +1,158 @@
+/* Font imports */
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@600&display=swap');
+
+/* 
+* Begin styling for html native elements
+*/
+
+* {
+	display: flex-box;
+	font-family: 'IBM Plex Mono';
+	font-weight: 300;
+	margin: 0;
+	padding: 0;
+}
+
+a {
+	color: inherit;
+	text-decoration: none;
+}
+
+a:hover {
+	text-decoration: underline;
+}
+
+h1 {
+	font-family: Inter;
+	font-size: 3rem;
+}
+
+h2 {
+	font-size: 2rem;
+}
+
+h3 {
+	font-size: 1.2rem;
+}
+
+html body {
```

The selector html body means the body element within html which is just the body element.  html is not needed.
```suggestion
body {
```